src/tiledimage.js

```diff
@@ -533,6 +549,32 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
                 OpenSeadragon.getElementPosition( this.viewer.element ));
     },
 
+    // private
+    // Convert rectangle in tiled image coordinates to viewport coordinates.
+    _tiledImageToViewportRectangle: function(rect) {
+        var scale = this._scaleSpring.current.value;
```

Comment:
Hmm... good question (on what the better name should be). Maybe it's just a matter of updating the comment to make it more explicit what you're talking about. Or you could use some variation on the phrase "point coordinate" as you used above. I'll let you know if I think of anything else. At any rate, probably not too big of a deal since it's private.

As for whether we should keep `_tiledImageToViewportRectangle`, what do you think the chances are we'll want it in the future?